tests/features/targets/import_targets.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Import targets directly to RSTUF SQL Database
+    As an admin,
+    Admin has deployed and bootstrapped RSTUF
+    Admin has an existent large number of targets (i.e.: 1,000,000 targets)
+    Admin wants to load targets in chunks to process them faster
+
+    Background:
+        Given admin has the RSTUF CLI in a host with access to RSTUF SQL Database
+        And admin has logged in with RSTUF CLI
+        And admin has the credentials to access RSTUF SQL Database
+        And admin has exported the targets info to CSV files (respecting CSV limits) with the format 'path;size;hash_type;hash;custom;'
```

Comment:
If you want to limit the chunks to up to `500 000` rows per CSV file (as you have mentioned in your comment suggestion here: https://github.com/vmware/repository-service-tuf/issues/188#issuecomment-1400576033) I think you want to mention it's not the CSV limits that should be respected, but our own threshold.

Reply:
I'm thinking about removing this limit. See my comment [above](https://github.com/vmware/repository-service-tuf/pull/218#discussion_r1090478041)